main.cpp

```diff
@@ -756,6 +788,23 @@ void DrumMachine::recPattern(int patternLength)
         std::cout << "Pattern recording with pattern length " << patternLength << std::endl;
 }
 
+
+Pattern DrumMachine::checkPatternPos(int pos, int start, int end)
+{
+    Pattern ptrn(0);
+
+    for(int s = start; s <= end; s++)
+    {
+        if(pos == s) {
+            ptrn.playHeadPos = pos;
+            return ptrn;
+        }
+    }
```

Comment:
curly braces go on their own line
can you add some std::cout statements inside the for() loop that print something out that shows your loop incrementing? 
- [ ] revise